maven-release-manager/src/main/java/org/apache/maven/shared/release/DefaultReleaseManager.java

```diff
@@ -560,15 +559,10 @@ private ReleaseDescriptorBuilder loadReleaseDescriptorBuilder( ReleaseDescriptor
     {
         try
         {
-            updateListener( listener, "verify-release-configuration", PHASE_START );
-            ReleaseDescriptorBuilder result = configStore.get().read( builder );
-            updateListener( listener, "verify-release-configuration", PHASE_END );
-            return result;
```

Comment:
Why is this information not necessary anymore?

Reply:
the log is not useful, and it's even confusing because it writes as a phase, but it's not = the initial intent of this whole Jira issue

Reply:
Agreed.